src/test/java/org/jabref/logic/importer/fileformat/ImporterTestEngine.java

```diff
@@ -0,0 +1,56 @@
+package org.jabref.logic.importer.fileformat;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.jabref.logic.bibtex.BibEntryAssert;
+import org.jabref.logic.importer.Importer;
+import org.jabref.model.entry.BibEntry;
+
+import org.junit.jupiter.api.Assertions;
+
+public class ImporterTestEngine {
+
+    private static final String TEST_RESOURCES = "src/test/resources/org/jabref/logic/importer/fileformat";
```

Comment:
Maybe better to use Paths.get(ImporterTestEngine.class).getResource("").toUri())

Reply:
I see you have the getPath already